companion/src/modeledit/setup.cpp

```diff
@@ -1348,11 +1356,29 @@ void SetupPanel::populateThrottleSourceCB()
   lock = false;
 }
 
+void SetupPanel::populateThrottleTrimSwitchCB()
+{
+  Board::Type board = firmware->getBoard();
+  lock = true;
+  ui->throttleTrimSwitch->clear();
+  int idx=0;
+  for (int i=0; i<getBoardCapability(board, Board::NumTrims); i++, idx++) {
+    QString trim = RawSource(SOURCE_TYPE_TRIM, i).toString(model, &generalSettings);
+    trim = (trim == "TrmR") ? "TrmT" : (trim == "TmrT") ? "TmrR" : trim;
```

Comment:
@dgatf can you explain this? It doesn't work with translations, and doesn't for English either (TrmT is there twice, TrmR is nowhere)

Reply:
Yes, is easy to explain... it is a typo. This is the line 1367 fixed:

`trim = (trim == "TrmR") ? "TrmT" : (trim == "TrmT") ? "TrmR" : trim;`

Please read the entire post to understand why R and T are interchanged...

_I've made the assumption that non used eeprom values are 0 (this assumption has been checked with several models on a X7 radio). So g_model.thrTrimSw will be 0 in existing models when updating opentx from previous releases. And the default switches order is RETA, so T has to go in first place (0) (this is safe as on previous releases throttle trim could only be used correctly on the default throttle switch). This is achieved by interchanging R (2) and T (0). And this is reason of the else if_

Fixed with #8141

Reply:
But `trim == "TmrT"` still won't work when companion is set say to French and you've got "TrmG" instead?

Reply:
You' re right. Fixed in last commit 4be12eb